docs/puzzles/cats.md

```diff
@@ -187,6 +200,51 @@ The CAT1 standard currently includes three example TAILs, though many more are p
+
+If a CAT achieves a certain level of financial success, its face value will be greater than its melt value, just like the $1 trillion coin would be worth more than the metal it was minted from. For example:
+
+* A meme token could trade for one-millionth of an XCH, or 1,000,000 mojos. The token is worth very little money, but it's still 1000 times more valuable than its melt value of 1000 mojos.
+* A dollar-backed stablecoin will have a face value of $1. Its melt value will be 1000 mojos.
+* A highly successful token could even sell for more than one XCH -- there are no rules preventing this from happening. Its melt value would still be 1000 mojos.
+
+One real-world analogy for these three cases is that you could start with a piece of metal and mint a coin worth a fraction of a cent, or $1, or $10,000, or really any other value. But no matter the face value, the melt value would always remain the same.
+
+**CAT retirement use cases and rules**
+
+Why would someone want to retire a CAT? Let's consider two cases, one involving the token's current owner and one involving its creator.
+
+If a CAT is not financially successful, its melt value could exceed its face value, in the same way that the metals that compose a US penny are worth more than one cent. In this case, it might make financial sense for the CAT's owner to retire the token by melting it into 1000 XCH mojos.
+
+However, it's important to keep in mind that a CAT's TAIL (and nothing else) decides the rules for retirement, _if_ it allows retirement at all. For example, our single-mint TAIL only works with a specific coin, so it does not allow retirement. CATs that use this TAIL can never be melted, no matter how small their face value.
+
+Our delegated TAIL leaves it entirely up to the CAT's creator whether -- and how -- retirement can happen. In many cases (for example, stablecoins and redemption tokens), retirement will be allowed, but only by the _creator_. In the case of a stablecoin, the creator may need to remove some tokens from circulation if backing funds are reduced. For redemption tokens, the owner may exchange their token with the creator for something of value. The token no longer has any face value, so its creator will retire it, in order to both take it out of circulation, and gain its melt value.
+
+In both of these cases, the CAT's creator must also be the current owner of the tokens in order to melt them.
+
+It's worth stressing this point. TAILs _can_ have nearly limitless functionality, but typically, if a TAIL allows retirement at all, two rules must be followed:
+
+1. The CAT's creator is the only person who can melt the tokens.
+2. The CAT's creator must be in possession of the tokens in order to melt them.
+
+There may be some use cases where the creator will want the owners to be able to retire their own tokens, but we expect these occasions to be rare.
```

Comment:
I don't know how necessarily rare it will be.  Maybe going from CAT to XCH, but there's a lot of cases when you might be allowed to melt a bunch of CAT for another type of CAT or for an NFT or something.  Maybe a small section about that would be good.

Reply:
I hadn't considered that. Thanks.